pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,99 @@
+// ------------------------------
+// edX Pattern Library: Components - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// ----------------------------
+// #SETTINGS
+// ----------------------------
+@mixin alert-base($alert-color) {
+    border-top: rem(4) solid $alert-color;
+    .alert-icon {
+        color: palette(grayscale, white);
+        background-color: $alert-color;
+    }
+}
+
+// ----------------------------
+// #GENERAL
+// ----------------------------
+.alert {
+    border: rem(1) solid palette(grayscale-cool, light);
+    border-radius: rem(3);
```

Comment:
Please make this value into a theming-friendly Sass variable that can be configured and overriden. See https://github.com/edx/ux-pattern-library/blob/master/pattern-library%2Fsass%2Fpatterns%2F_lists.scss#L22 as an example of how we've done it before.